Apps/W1/HybridGP/app/src/Migration/Support/WizardIntegration.codeunit.al

```diff
@@ -74,6 +75,7 @@ codeunit 4035 "Wizard Integration"
         DataSyncStatus.ParsePosting();
         HelperFunctions.PostGLTransactions();
         HelperFunctions.SetProcessesRunning(false);
+        HybridCloudManagement.CreateAndScheduleBackgroundJob(Codeunit::"GP Populate Hist. Tables", 'Migrate GP Historical Snapshot');
```

Comment:
This isn't considered pass\fail criteria for the GP migration? How does one know if this succeeds or not?

Reply:
Or is the background task which is created still running under the 'cannot log into company being migrated' context and the migration isn't considered complete until that background job is done too?

Reply:
This could use more discussion. Nikola's recommendation was to move this load logic to a background job because it could take a very long time to finish.